p3/templatetags/meta.py

```diff
@@ -16,12 +16,12 @@ def render_meta(context, title=""):
 
     description = ""
     page_title = (
-        f"{title}EuroPython 2020 Online &middot; 23-26 July 2020"
+        f"{title}EuroPython 2021 Online &middot; July 26 - Aug. 1, 2021"
```

Comment:
This should read "Jul 26 - Aug 1, 2021" to be consistent with our other mentions of the dates (corresponds to Wikipedia date format standards).

Reply:
Fixed